Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Ride.hs

```diff
@@ -134,7 +136,21 @@ newtype EndRideReq = EndRideReq
 instance HideSecrets EndRideReq where
   hideSecrets = identity
 
----------------------------------------------------------
+--------------------------- MultipleRideEndAPI -----------------------------
+type MultipleRideEndAPI =
+  "end"
+    :> ReqBody '[JSON] MultipleRideEndReq
+    :> Post '[JSON] [APISuccess]
```

Comment:
`[APISuccess]` this return type make no sense. It would be more informative to return list of rides, that ended successfully